validator_client/src/cli.rs

```diff
@@ -1,478 +1,466 @@
-                .action(ArgAction::SetTrue)
-                .help_heading(FLAG_HEADER)
-                .display_order(0)
-        )
-        /* Prometheus metrics HTTP server related arguments */
-        .arg(
-            Arg::new("metrics")
-                .long("metrics")
-                .help("Enable the Prometheus metrics HTTP server. Disabled by default.")
-                .action(ArgAction::SetTrue)
-                .help_heading(FLAG_HEADER)
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("metrics-address")
-                .long("metrics-address")
-                .requires("metrics")
-                .value_name("ADDRESS")
-                .help("Set the listen address for the Prometheus metrics HTTP server.")
-                .default_value_if("metrics", ArgPredicate::IsPresent, "127.0.0.1")
-                .action(ArgAction::Set)
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("metrics-port")
-                .long("metrics-port")
-                .requires("metrics")
-                .value_name("PORT")
-                .help("Set the listen TCP port for the Prometheus metrics HTTP server.")
-                .default_value_if("metrics", ArgPredicate::IsPresent, "5064")
-                .action(ArgAction::Set)
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("metrics-allow-origin")
-                .long("metrics-allow-origin")
-                .requires("metrics")
-                .value_name("ORIGIN")
-                .help("Set the value of the Access-Control-Allow-Origin response HTTP header. \
+    styles = get_color_style(),
+    next_line_help = true,
+    term_width = 80,
+    disable_help_flag = true,
+    disable_help_subcommand = true,
+    display_order = 0,
+)]
+pub struct ValidatorClient {
+    #[clap(
+        long,
+        value_name = "NETWORK_ADDRESSES",
+        value_delimiter = ',',
+        help = "Comma-separated addresses to one or more beacon node HTTP APIs. \
+                Default is http://localhost:5052.",
+        display_order = 0
+    )]
+    pub beacon_nodes: Option<Vec<String>>,
+
+    #[clap(
+        long,
+        value_name = "NETWORK_ADDRESSES",
+        value_delimiter = ',',
+        help = "Comma-separated addresses to one or more beacon node HTTP APIs. \
+                These specify nodes that are used to send beacon block proposals. \
+                A failure will revert back to the standard beacon nodes specified in --beacon-nodes.",
+        display_order = 0
+    )]
+    pub proposer_nodes: Option<Vec<String>>,
+
+    #[clap(
+        long,
+        value_name = "API_TOPICS",
+        value_delimiter = ',',
+        help = "Comma-separated list of beacon API topics to broadcast to all beacon nodes. \
+                Possible values are: none, attestations, blocks, subscriptions, \
+                sync-committee. Default (when flag is omitted) is to broadcast \
```

Comment:
Possible values are printed twice in help, this can now be removed:

```suggestion
        help = "Comma-separated list of beacon API topics to broadcast to all beacon nodes. \
                Default (when flag is omitted) is to broadcast \
```